src/__tests__/__snapshots__/settingsOverview.test.js.snap

```diff
@@ -334,14 +334,6 @@ exports[`MyComponent should render Locked SettingsOverview 1`] = `
         disabled={false}
         leftText="Ufo"
       />
-      <RowRoute
```

Comment:
Did you intend to remove these lines?

Reply:
in the test no, however when the tests were failing from the changing of the icon name , the snap shot was regenerated removing these lines.. you can see there are no corresponding changes in the actual file.